package.json

```diff
@@ -27,6 +27,7 @@
     "@react-native-community/eslint-config": "^3.2.0",
     "@react-native-community/netinfo": "^9.3.7",
     "@react-navigation/bottom-tabs": "^6.5.5",
+    "@react-navigation/material-bottom-tabs": "^6.2.15",
```

Comment:
Can the `@react-navigation/bottom-tabs` dep above this be removed, or is it still needed?

Reply:
Yep, removed.